lib/device_impl.toit

```diff
@@ -64,3 +73,42 @@ class Gnss_:
 
 get_mac_address_:
   #primitive.esp32.get_mac_address
+
+class Volume_:
+  name_ ::= ?
+  read_only_ ::= ?
+
+  constructor.init_ .name_ .read_only_:
+
+  from name:
+    return KeyValue_ this name
+
+class KeyValue_:
+  group_ ::= ?
```

Comment:
Add type?

Reply:
This is a resource proxy, so there is no good type for it.